file/file-alfresco/src/test/java/uk/gov/justice/services/file/alfresco/requester/AlfrescoFileRequesterIT.java

```diff
@@ -26,93 +25,86 @@
 
 public class AlfrescoFileRequesterIT {
 
+    public static final String BASE_PATH = "http://localhost:%d/alfresco";
     private static int PORT = getNextAvailablePort();
-    private static final String BASE_PATH = format("http://localhost:%d/alfresco", PORT);
     public static final String ALFRESCO_WORKSPACE_PATH = "/service/api/node/content/workspace/SpacesStore/";
 
-    private static AlfrescoFileRequester fileRequester;
+    private static FileRequester fileRequester;
 
     @Rule
     public WireMockRule wireMock = new WireMockRule(PORT);
 
     @BeforeClass
     public static void beforeClass() {
-        fileRequester = new AlfrescoFileRequester();
-        fileRequester.alfrescoWorkspacePath = ALFRESCO_WORKSPACE_PATH;
-        fileRequester.alfrescoReadUser = "user1234";
-        fileRequester.restClient = new AlfrescoRestClient();
-        fileRequester.restClient.alfrescoBaseUri = BASE_PATH;
-        fileRequester.restClient.proxyType = "none";
+        final int port = PORT;
+        fileRequester = alfrescoFileRequesterWith(basePathWithPort(port));
     }
 
     @Test
     public void shouldRequestFileFromAlfresco() throws Exception {
         final String fileId = randomUUID().toString();
         final String mimeType = "text/plain";
         final String fileName = "file.txt";
-        final FileData fileData = new FileData(fileId, mimeType);
 
-        fileRequester.request(fileData , fileName);
+        fileRequester.request(fileId, mimeType, fileName);
 
-        verify(getRequestedFor(urlEqualTo("/alfresco" + ALFRESCO_WORKSPACE_PATH + fileId + "/content/"+ fileName))
-               .withHeader(ID, equalTo("user1234")));
+        verify(getRequestedFor(urlEqualTo("/alfresco" + ALFRESCO_WORKSPACE_PATH + fileId + "/content/" + fileName))
+                .withHeader("cppuid", equalTo("user1234")));
     }
 
     @Test
-    public void shouldReturnResponseFromAlfresco(){
+    public void shouldReturnResponseFromAlfresco() {
         final String fileId = randomUUID().toString();
         final String mimeType = "text/plain";
```

Comment:
I'm not sure if it's a good practice to use constants in tests.
I think it makes tests more readable if you directly declare and assign variables in each individual test case which you later use in assertions